clang/test/FixIt/format.cpp

```diff
@@ -1,5 +1,6 @@
 // RUN: %clang_cc1 -fsyntax-only -verify -Wformat %s
-// RUN: %clang_cc1 -fsyntax-only -fdiagnostics-parseable-fixits -Wformat %s 2>&1 | FileCheck %s
+// RUN: %clang_cc1 -fsyntax-only -fdiagnostics-parseable-fixits -Wformat-pedantic %s 2>&1 | FileCheck %s
```

Comment:
With the RUN: line including `-Wformat-pedantic`, do we need the rest of the changes (adding `expected-no-diagnostics` and the removal of `expected-warning`s)?